src/utils/json.ts

```diff
@@ -1,11 +1,15 @@
 import Json from 'json-bigint';
 
-const { parse, stringify } = Json({
-  alwaysParseAsBig: true,
-  useNativeBigInt: true,
-  protoAction: 'preserve',
-  constructorAction: 'preserve',
-});
+const json = (alwaysParseAsBig: boolean) => {
+  return Json({
+    alwaysParseAsBig,
+    useNativeBigInt: true,
+    protoAction: 'preserve',
+    constructorAction: 'preserve',
+  });
+};
+
+export const { parse, stringify } = json(false);
+export const { parse: parseAsBig, stringify: stringifyAsBig } = json(true);
```

Comment:
should be `parseAlwaysAsBig`

as the normal function can also parse as big when needed